zendesk/resource_zendesk_ticket_field.go

```diff
@@ -168,6 +168,124 @@ func resourceZendeskTicketField() *schema.Resource {
 	}
 }
 
+func marshalTicketField(d identifiableGetterSetter) (client.TicketField, error) {
```

Comment:
Is this unmarshal?
Sorry, I'm a little bit confused.

- `TicketField` => `ResourceData`
- `ResourceData` => `TicketField`

Which is marshal?

Reply:
yeah probably should change the name here. this is going resource data => ticketfield

Reply:
OK, thanks.